### PR TITLE
[FIX] Change recommended anonymization date from 1900 to 1925

### PR DESCRIPTION
**Pull request by alexrockhill**
Fixes issue https://github.com/bids-standard/bids-specification/issues/360

### CODE REVIEW

File: src/99-appendices/01-contributors.md [hunk 1]
```diff
@@ -132,6 +132,7 @@ your name is not listed, please add it.
 -   Pradeep Reddy Raamana 💻🔧
 -   Vasudev Raguram 💻🎨📖🔧
 -   Nick F. Ramsey 📖
+-   Alex Rockhill
```
Comment (effigies) [inline src/99-appendices/01-contributors.md]: Based on this and MNE-BIDS, it seems like the following codes are applicable:

```suggestion
-   Alex Rockhill 📖🔧
```

Feel free to add others that you feel apply.

File: src/03-modality-agnostic-files.md [hunk 2]
```diff
@@ -185,7 +185,7 @@ one subject should be shifted by a randomly chosen (but common across all runs
 etc.) number of days. This way relative timing would be preserved, but chances
 of identifying a person based on the date and time of their scan would be
 decreased. Dates that are shifted for anonymization purposes should be set to a
-year 1900 or earlier to clearly distinguish them from unmodified data. Shifting
+year 1925 or earlier to clearly distinguish them from unmodified data. Shifting
 dates is recommended, but not required.
```
Comment (effigies) [inline src/03-modality-agnostic-files.md]: While we're here...

```suggestion
dates is RECOMMENDED, but not required.
```

**Comment by alexrockhill** [comment]
Checking if anyone has time to review this PR

**Comment by alexrockhill** [comment]
Ok to merge?

**Comment by sappelhoff** [comment]
Thanks @alexrockhill

**Pull request merged**